client/reducers/socialClubReducers.js

```diff
@@ -27,7 +27,13 @@ export function socialClubs(state = initialState.socialClubs, action) {
 export function socialClub(state = initialState.socialClub, action) {
   switch(action.type){
     case GET_CLUB:
-      return action.club;
+      return action.payload;
+    case CREATE_EVENT:
+      return Object.assign({}, state, {
+        events: state.events.concat(action.payload)
+      });
+    case SIGN_OUT:
+      return initialState.socialClub;
```

Comment:
This should return state instead, since we have specified what we want to work with when passing the argument into this reducer function.

Reply:
`state` at a particular time is the current state which might not be the same as the initial state. On signout, I need the state to return to the initial (empty) state.